docs/pages/api-docs/data-grid.md

```diff
@@ -22,6 +22,7 @@ import { DataGrid } from '@material-ui/data-grid';
 | <span class="prop-name">columnTypes</span> | <span class="prop-type">ColumnTypesRecord</span> |   | Extend native column types with your new column types. |
 | <span class="prop-name">components</span> | <span class="prop-type">GridComponentOverridesProp</span> |   | Overrideable components. |
 | <span class="prop-name">density</span> | <span class="prop-type">Density</span> | standard | Sets the density of the grid. |
+| <span class="prop-name">disableColumnMenu</span> | <span class="prop-type">boolean</span> | false | If `true`, the column menu will be disabled. |
```

Comment:
1. We use the present tense
```suggestion
| <span class="prop-name">disableColumnMenu</span> | <span class="prop-type">boolean</span> | false | If `true`, the column menu is disabled. |
```

2. We need to update the two other descriptions of the prop `disableColumnMenu?: boolean;` to match.

Reply:
1. is point number 4 of https://trello.com/c/1Q7hqXHu/2714-technical-documentation-guidelines.

Reply:
What about point 2?

Reply:
There is only the `x-grid.md` file where the prop is mentioned, just updated it.

Reply:
> We need to update the two other descriptions of the prop

I meant in the codebase, not in the documentation

Reply:
One bonus place, inside the `colDef.ts` there is another prop called `disableColumnMenu` that is specific for that column. I've updated all of them.